javalanglib/src/main/scala/java/lang/Double.scala

```diff
@@ -82,6 +84,61 @@ object Double {
   @inline def toString(d: scala.Double): String =
     "" + d
 
+  def toHexString(d: scala.Double): String = {
```

Comment:
[JavaDoc of `Double.toHexString`](https://docs.oracle.com/javase/8/docs/api/java/lang/Double.html#toHexString-double-)